docs/Makefile

```diff
@@ -60,6 +61,9 @@ rest-api: source/_static/rest-api/index.html
 source/_static/rest-api/index.html: rest-api.yml node_modules
 	npm run rest-api
 
+metrics:
+	python3 generate-metrics.py
+
 html: rest-api
```

Comment:
I've learned that the `rest-api` here is a _prerequisite_ for the `html` Makefile _target_ (https://makefiletutorial.com/).

What is the relationship between the `metrics` target and `html` target? Is metrics needed to run for the html to run properly? I think so, because in docs/source/index.rst a monitoring/index is referenced, which is a document which in turn reference `_gen/metrics`.

@rajat404, this is my current understanding:
1. generate-metrics.py doesn't depend on the file source/_static/rest-api/index.html
2. doing `make html` now depends on `_gen/metrics` through `doc/source/index.rst` and docs/source/monitoring/index.rst`

I'll see if I can mimic this system for the rest-api documentation.

```Makefile
rest-api: source/_static/rest-api/index.html

source/_static/rest-api/index.html: rest-api.yml node_modules
	npm run rest-api

html: rest-api
```

Reply:
I pushed a commit to let `make html` depend on generating metrics, just like `make html` depends on generating the rest-api documentation first and installing npm packages as well.